packages/fiori/src/ViewSettingsDialog.ts

```diff
@@ -202,6 +215,26 @@ class ViewSettingsDialog extends UI5Element {
 	@property({ type: Boolean })
 	sortDescending!: boolean;
 
+	/**
+	 * Defines the ID or DOM Reference of the element at which the dialog is shown.
+	 * When using this attribute in a declarative way, you must only use the `id` (as a string) of the element at which you want to show the popover.
+	 * You can only set the `opener` attribute to a DOM Reference when using JavaScript.
+	 * @public
+	 * @default ""
+	 * @since 1.10.0
+	 */
+	@property({ validator: DOMReference, defaultValue: "" })
+	opener!: HTMLElement | string;
```

Comment:
Do we need `opener`, I think popovers need it, but dialogs not as they are shown at the same place, not anchored to the opener as the popovers

Reply:
Now that you say that, I agree that it's not needed.  I am removing it.